sycl/doc/UsersManual.md

```diff
@@ -225,6 +225,14 @@ and not recommended to use in production environment.
 
     NOTE: This option is currently only supported on Linux.
 
+**`-fsycl-force-target=<T>`**
+
+    When used along with '-fsycl-targets', force the device object being
+    unbundled to match the <T> given.  This allows the user to override the
```

Comment:
```suggestion
    unbundled to match the target <T> given.  This allows the user to override the
```
Makes it clearer what the later mention of "forced target" refers to.

Reply:
Done - thanks